### ISSUE TITLE
`TypeGuard` without a type argument cannot be used in an expression

### ISSUE THREAD

**Issue by bryanforbes** [microsoft/pyright#2986]
**Describe the bug**
Given the following (admittedly contrived code):

```python
from __future__ import annotations

from typing import Any, get_origin, get_type_hints
from typing_extensions import TypeGuard


def is_thing(obj: Any) -> TypeGuard[int]:
    return isinstance(obj, int)


annotations = get_type_hints(is_thing)

if get_origin(annotations['return']) is TypeGuard:
    print('here')
```

An error is generated for line 13: `Expected a single type argument after "TypeGuard"`.

**Expected behavior**
No error should be generated.

**VS Code extension or command-line**
Are you running pyright as a VS Code extension or a command-line tool?
Command line

Which version?
1.1.217

**Comment by erictraut** [comment]
This will be addressed in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.218, which I just published. It will also be included in the next release of pylance.
